### DISCUSSION TITLE
How to setup SMTP on Postal

### DISCUSSION THREAD

**Discussion by GrooveDevops** [postalserver/postal#1710] [Help with using Postal]
Hi There,

I have trying to setup SMTP on postal but don't know where to start

**Comment by willpower232** [comment]
how far have you gotten? have you added a domain and credential?

**Reply by GrooveDevops** [reply]
added domain Newley purchased domain  

![image](https://user-images.githubusercontent.com/77006816/141752002-da1db3bd-ddbc-4514-9dac-072798d3913e.png)

**Reply by willpower232** [reply]
have you added a Mail Server on that page?

**Reply by GrooveDevops** [reply]
But Postal is email server right !!!!!!!! 

added same domain as we using for postal......... is this right ???
![image](https://user-images.githubusercontent.com/77006816/141754021-a54645f3-cedb-493a-a517-11984c58adab.png)

Accordingly, I added cred here

![image](https://user-images.githubusercontent.com/77006816/141754754-aa7b7a8d-9448-40f9-9a0c-9bf104c0c1cd.png)

**Reply by willpower232** [reply]
you can now click on "Read more about sending outgoing e-mails" and add those details to whatever you want to send an email.

**Reply by GrooveDevops** [reply]
Yeah I can see the below details there but how can I trigger an email 

I checked there and found below interaction and try to test same but did workout 
![image](https://user-images.githubusercontent.com/77006816/141928164-3869b00b-c210-43ce-be31-cb09d63ae1ee.png)

My question is what user should I use for sending an email where is my domain urva.app I didn't see any option to create users

**Reply by willpower232** [reply]
Postals purpose is to enable things to send emails like website CMSes or even printers. It doesn't have any concept of mailboxes or long term email storage.

**Reply by GrooveDevops** [reply]
So you mean we should have an email provider in place before setup postal ????? 
And we can use Postal as a relay !!!!!

**Reply by willpower232** [reply]
If you want to then you could do that but if you already have an email provider then you likely don't need Postal
